README.md

```diff
@@ -299,7 +302,7 @@ The following items give the `DATATYPE`, its description, and the default value
         IF <condition> THEN
             <statements>
         ENDIF
-
+        
```

Comment:
Please remove all of these useless spaces!

Reply:
OK